templates/layouts/application.html

```diff
@@ -9,11 +9,16 @@
   </head>
 
   <body>
-    <main class="app-main">
-      <div class="app-main__content">
-        {{ template "content" . }}
-      </div>
-    </main>
+    <div class="app">
+      {{ include "partials/navbar" }}
```

Comment:
Because I need `flex display` for navbar and app but not for auth.
if wrap everything in body tag and `display flex` in the body, this will effect the auth layout also.

What do you think?

Reply:
Actually, I understand the issue now and it also relates to another issue with screen stylesheets.

In the layout HTML, the body tag just have dynamic class names based on the controller/route name:

```
<body class="register show">
<body class="keyword show">
```

So that it can be used in screen stylesheets:

```
body.keyword .form-keyword-upload {

}

body.keyword .list-keyword {

}
```

This way the styles are only scoped to each page/screen.

In rails, that's what we use in the layout ERB/SLIM files:

```
body class="#{body_class_name}"
```

Which maps to this helper:

https://github.com/nimblehq/mia-web-backup/blob/64c275874fa085455ebb3bb4d1f7c09bf80da49f/app/helpers/css_helper.rb#L1-L11

So you need to find a similar way for Gin

Reply:
Add controller class name to body in 543c36deb803b7d8024c6cd43fe795d26ed3c039
Update stylesheet in 6cdf6b77decf45ff5879d3fd7a8714a18b0b2c51